starter-code/iteration1/src/app/app.component.ts

```diff
@@ -6,5 +6,11 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'app works!';
+  title: string = 'app works!';
+  description1: string = 'holi';
+  description2: string = 'caracoli';
+  description3: string = 'pocholi';
+  imgURL1: string = 'https://salaspequenas.com/wp-content/uploads/2015/11/mm.jpg';
+  imgURL2: string = 'http://es.fordesigner.com/pic/zip/20095219572112477801.jpg';
+  imgURL3: string = 'https://www.revistaesfinge.com/media/k2/items/cache/x8cba5d214c5f756cb3dfba3d009c9a8f_L.jpg.pagespeed.ic.AARsQpzp4R.jpg';
 }
```

Comment:
holi pocholi! 

Bien hecha la interfaz y la aplicación al HTML. 

Aún así, es más correcto definir la interfaz tal y como has hecho, pero asignar los valores en el constructor, de forma que esto ocurra al instanciar la clase.

Ej: 
export class AppComponent {
    description1: string;

    constructor() {
        this.description1 = 'ola k ase';
    }
}